src/util/adb.js

```diff
@@ -86,13 +86,31 @@ export default class ADBUtils {
     return devices.map((dev) => dev.id);
   }
 
+  async getCurrentUser(deviceId) {
+    log.debug(`Retreiving current user on ${deviceId}`);
+
+    const currentUser = await this.runShellCommand(deviceId, [
+      'am',
+      'get-current-user',
+    ]);
+
+    const userId = parseInt(currentUser.trim());
+    if (isNaN(userId)) {
+      throw new WebExtError(`Unable to retreive current user on ${deviceId}`);
```

Comment:
> Codecov complains about the new exception line testing the result of (`am get-current-user`), but that's extremally unlikely condition as the command result is tested earlier, so perhaps does not require an additional test.

codecov is correct. There is no test case that confirms what happens when `adb shell am get-current-user` returns an unexpected value.

Reply:
Yes, deemed that testcase irrelevant given it would either fail or return a user ID.

Reply:
Fixed